From: Rob Browning <rlb@defaultvalue.org>
Date: Sun, 5 Feb 2023 07:04:14 +0900
Subject: debian-site-init-el

* Emacs will prefer /usr/share/info/emacs22 over /usr/share/info.
  Patch: debian-site-init-el.diff

  The value of Info-default-directory-list has been augmented via
  site-init.el to include /usr/share/info/emacs-22 before
  /usr/share/info.
---
 lisp/site-init.el | 11 +++++++++++
 1 file changed, 11 insertions(+)
 create mode 100644 lisp/site-init.el

diff --git a/lisp/site-init.el b/lisp/site-init.el
new file mode 100644
index 0000000..7e3af35
--- /dev/null
+++ b/lisp/site-init.el
@@ -0,0 +1,11 @@
+
+;; Add the flavor specific directory for Debian.
+(setq Info-default-directory-list
+      (let ((result nil)
+            (found-match nil))
+        (dolist (elt Info-default-directory-list (nreverse result))
+          (if (and (not found-match) (equal elt "/usr/share/info/"))
+              (progn
+                (push "/usr/share/info/emacs-22" result)
+                (setq found-match t)))
+          (push elt result))))
